### PR TITLE
add tokeo wallet to showcase section

### PR DESCRIPTION
**Pull request by thenic95**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Showcase Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Showcase addition

* Title: Tokeo
* Description: Tokeo is a next-generation smart mobile wallet designed for Cardano and beyond, featuring secure, intuitive trading, native token swaps, NFT interactions, and cross-chain support with enhanced personalization and security standards.
* Website: https://tokeopay.io/
* Source: /
* Tags:
  * `nftsupport`
  * `wallet`

### CODE REVIEW

File: src/data/showcases.js [hunk 1]
```diff
@@ -1163,6 +1163,15 @@ export const Showcases = [
     source: null,
     tags: ["favorite", "educational"],
   },
+  {
+  title: "Tokeo",
+  description:
+    "Tokeo is a next-generation smart mobile wallet designed for Cardano and beyond, featuring secure, intuitive trading, native token swaps, NFT interactions, and cross-chain support with enhanced personalization and security standards.",
```
Comment (rdlrt) [inline src/data/showcases.js:1169]: > next-generation smart mobile wallet

?

**Comment by rdlrt** [comment]
'launching soon'
image

**Comment by thenic95** [comment]
@rdlrt The wallet is currently available on iOS App store link: https://apps.apple.com/au/app/tokeo-wallet/id6476824031

According to the following [tweet](https://x.com/tokeopay/status/1807791291830313259) Tokeo is only available in the following countries atm:

> Albania, Australia, Austria, Belarus, Belgium, Bosnia and Herzegovina, Bulgaria, Canada, Croatia, Cyprus, Czech Republic, Denmark, Estonia, Finland, France, Germany, Greece, Hungary, Iceland, Ireland, Italy, Japan, Kenya, Kosovo, Latvia, Lithuania, Luxembourg, Malta, Moldova, Montenegro, Netherlands, New Zealand, North Macedonia, Norway, Poland, Portugal, Romania, Russia, Serbia, Slovakia, Slovenia, Spain, Sweden, Switzerland, Türkiye, Ukraine, United States.

File: src/data/showcases.js (hunk 1 above)
Comment (thenic95) [reply src/data/showcases.js]: Good point, I just updated the description.

**Review by rphair — COMMENTED** [review]
I'm unwilling to approve this yet, given the reservations pointed out already above.  The first thing I did was look for GitHub addresses of developers to tag in this comment (yes I know it's not an open source app, and doesn't have to be for approval), following every single link on the [Team page](https://tokeo.gitbook.io/tokeo/tokeo/tokeo/our-team), and found nothing.  This collective link profile could be used to promote a spread of social influence but not to establish any accountability.

On that note, the [Support page](https://tokeo.gitbook.io/tokeo/tokeo/support) is currently another "Coming Soon"... the page empty except for that single statement.  An email address is on the previous page in the navigation but doesn't say what people who email it can expect.

Finally the page lacks an explanation of **why** it is only available in certain countries.  Hopefully this was simply some web designer / marketer's decision to leave that off, rather than a calculated decision to leave people in the dark about potential regulatory issues which could leave users without support for created wallets and perhaps leave their assets tainted or frozen.

I won't red-flag this since I trust @thenic95's skills and reputation and am sure there are valid reasons for adding this.  But I couldn't directly support this in the Showcase without a response from the developers about the apparent unreadiness and potential problems.

**Comment by rdlrt** [comment]
> @rdlrt The wallet is currently available on iOS App store link: https://apps.apple.com/au/app/tokeo-wallet/id6476824031

Indeed, but listing something in showcase that's not fully ready (be it website having 'coming soon' or missing play store link while mentioning support for android) does not fit well for me, especially when it comes to non-verifiably-open-source wallets - as they deal with end user funds and dev portal being hosted on a cardano.org domain 🙂

**Comment by thenic95** [comment]
Thank you, @rdlrt and @rphair, for your input. For now, it's best to close this PR and revisit adding the Tokeo Wallet at a later stage.

Closed-source wallets (e.g., VESPR) should also be eligible for the showcase section. However, in these cases, we need to apply a higher level of scrutiny and caution on behalf of the Cardano community.

**Comment by rdlrt** [comment]
> Closed-source wallets (e.g., VESPR) should also be eligible for the showcase section. 

Absolutely, the difference with VESPR is it's well reputation, adoption , reachability to devs while all the functionalities on their website has been live - so there isnt a similar concern to add them while being close-sourced 🙂

**Comment by katomm** [comment]
Yes and: to be fair, it took a while until VESPR wallet was merged.

**Pull request closed without merging**